boot/zephyr/Kconfig

```diff
@@ -429,7 +429,7 @@ config BOOT_SERIAL_DETECT_PIN
 	default 6 if BOARD_NRF9160DK_NRF9160
 	default 11 if BOARD_NRF52840DK_NRF52840
 	default 13 if BOARD_NRF52DK_NRF52832
-	default 23 if BOARD_NRF5340_DK_NRF5340_CPUAPP || BOARD_NRF5340_DK_NRF5340_CPUAPPNS
+	default 23 if BOARD_NRF5340PDK_NRF5340_CPUAPP || BOARD_NRF5340PDK_NRF5340_CPUAPPNS
```

Comment:
And what with BOARD_NRF5340**DK**_NRF5340_CPUAPP ?

Reply:
> And what with BOARD_NRF5340**DK**_NRF5340_CPUAPP ?

I don't understand your question. All I did was upstream a patch we had downstream.

Reply:
I guess we need both, @nvlsianpu

Reply:
yeap